### PR TITLE
[DEV] Working on `Sucuri-API` Modularization

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey @Aryan-sharma11 and @abhi18av, I tried to optimize the utilization of this module, and if the overall structure is good for both of you, this module should be ready for it first release!! v1.0.0 will export 2 functions, and you just need to say `Start-SucuriServer` to have the server working.

The overall content of this PR: 
1. Tests for the modularization
2. turning Start-ApiServet into a Function that can be exported by the module

Please, I'd like to hear your thoughts, and I'd also appreciate if you could run the tests and check if it's reproducible there.

This Pull-Request is related to #8 and would conclude the Modularization Step on #6 
Kindly, Davi

### CODE REVIEW

File: Sucuri-API/Public/Invoke-SucuriServer.ps1 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Function Invoke-SucuriServer{
+
+    Import-Module Pode
+    $ScriptPath = "$ENV:SucuriPath/get-version.ps1"
+    # Starting pode server
+
+    Start-PodeServer{
+
```
Comment (abhi18av) [inline Sucuri-API/Public/Invoke-SucuriServer.ps1:8]: @Mxrcon , this is the circular dependency problem I mentioned earlier :)

Comment (Mxrcon) [reply Sucuri-API/Public/Invoke-SucuriServer.ps1:8]: Sure, I'll work on that

Comment (abhi18av) [reply Sucuri-API/Public/Invoke-SucuriServer.ps1]: Thanks Davi 😊

**Review by abhi18av — APPROVED** [review]
Looks good to merge ✅ 

Thanks @Mxrcon !

**Review by Aryan-sharma11 — COMMENTED** [review]
Looks good!!

**Pull request merged**